gravity.sh

```diff
@@ -549,10 +549,15 @@ parseList() {
   # define valid domain patterns
   # no need to include uppercase letters, as we convert to lowercase in gravity_ParseFileIntoDomains() already
   # adapted from https://stackoverflow.com/a/30007882
-  # supported ABP style: ||subdomain.domain.tlp^
 
-  valid_domain_pattern="([a-z0-9]([a-z0-9_-]{0,61}[a-z0-9]){0,1}\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]"
-  abp_domain_pattern="\|\|${valid_domain_pattern}\^"
+  TLD_pattern="[a-z0-9][a-z0-9-]{0,61}[a-z0-9]"
+  subdomain_pattern="([a-z0-9]([a-z0-9_-]{0,61}[a-z0-9]){0,1}\.)"
+
+  valid_domain_pattern="${subdomain_pattern}+${TLD_pattern}"
+
+  # supported ABP style: ||subdomain.domain.tld^
+  # Subdomain is optional for ABP style, allowing TLD blocking: ||tld^
```

Comment:
```suggestion
  # Subdomain/domain is optional for ABP style, allowing TLD blocking: ||tld^
```

Reply:
I will change the comment to:

```
# subdomain_pattern is optional for ABP style, allowing TLD blocking: ||tld^
```

What do you think?

Reply:
Good